index.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
-const remark = require('remark');
-const remarkLint = require('remark-lint');
+const deepAssign = require('deep-assign');
```

Comment:
Use `lodash.merge` instead. `deep-assign` is buggy and I haven't had time to fix it.